client/src/components/classesPage/ClassEditView.js

```diff
@@ -21,32 +28,71 @@ import axios from 'axios';
 
 const styles = theme => ({
   wrapper: {
-    marginTop: theme.spacing.unit * 6,
-    display: 'flex',
-    flexDirection: 'column',
     border: `1px solid ${theme.palette.secondary.main}`,
     ...theme.mixins.gutters(),
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    justifyContent: 'space-between',
     alignItems: 'center',
+    paddingTop: theme.spacing.unit * 4,
+    paddingBottom: theme.spacing.unit * 8,
+    marginTop: theme.spacing.unit * 6,
+    marginBottom: theme.spacing.unit * 4,
     color: theme.palette.primary.contrastText,
     background: theme.palette.primary.dark,
-    [theme.breakpoints.down('sm')]: {
-      width: '80%'
+    [theme.breakpoints.only('sm')]: {
+      width: '60vw'
     },
```

Comment:
I got to get better with breakpoints. This page is super responsive and looks great at all sizes! :fire: